### DISCUSSION TITLE
Playback thru video hardware to external monitor (i.e Blackmagic Design Decklink card)

### DISCUSSION THREAD

**Discussion by tordenver** [videovillage/community#119] [Ideas]
I have a feature request. It would be nice if we could use the system hardware setup to send video to external monitor like my Decklink 4K Extreme card.

Thanks for considering.

**Comment by gregcotten** [comment]
Hi! Yes, this is definitely on our roadmap. Probably just video first, then properly synced audio at some point down the road.

**Comment by tordenver** [comment]
Thanks Greg.

David Baud, CSI
Colorist & Finishing Editor
––––––––––––––––––––––––––––––––
KOSMOS PRODUCTIONS ⁝ Denver - Paris
web: www.kosmos-productions.com <http://www.kosmos-productions.com/>
vimeo: davidbaud <https://vimeo.com/davidbaud>
email: david@kosmos-productions.com <mailto:david@kosmos-productions.com>
studio: +1.720.242.6238
cell: +1.303.548.5516
––––––––––––––––––––––––––––––––
member of SMPTE, CFVA & FACC
DaVinci Resolve Certified Trainer
Colorist Society International

**Comment by tordenver** [comment]
Greg - if you decide to add this feature to Screen I would highly recommend to implement the option to “release the hardware" when quitting the application or having it in the background… for a long time neither DaVinci Resolve or Adobe Suite had this option and it was a pain to have to restart your computer just to be able to switch applications.

Thanks for listening!

David

**Comment by wilg** [comment]
Duplicate of https://github.com/orgs/videovillage/discussions/94
